.github/workflows/ci.yml

```diff
@@ -52,7 +52,8 @@ jobs:
           composer --version
 
       - name: Build Admin feature config
-        run: pnpm build:feature-config --filter=woocommerce
+        working-directory: plugins/woocommerce
```

Comment:
These top level non-recursive commands don't really work the way they used to anymore (if the script is not present at root it fails).

Its simpler to just run them at the directory they are defined.